update read me file for Variable, vault, vaultGraphQl, version, webApi modules (#31474)
magento/magento2: app/code/Magento/Version/README.md

# Magento_Version module

> The Magento_Version module allows to get Magento version and edition by HTTP GET request.

Reviewer: ```suggestion
The Magento_Version module allows retrieval of the Magento version and edition using HTTP GET requests.
```
